framework/i18n/I18N.php

```diff
@@ -109,7 +109,7 @@ public function format($message, $params, $language)
             return $message;
         }
 
-        if (preg_match('~{\s*[\d\w]+\s*,~u', $message)) {
+        if (preg_match('~{\s*[\w]+\s*,~u', $message)) {
```

Comment:
Can be confusing, but it still [the same](https://3v4l.org/vDpse).
`\d\w` is a greedy matching because `\w ==  [A-Za-z0-9_]` shares `0-9` with `\d`.